helm/adelphi/values.yaml

```diff
@@ -58,6 +58,7 @@ cql_schema: |-
     value text,
     PRIMARY KEY (key)
   );
+gemini_schema_keyspace: testkeyspace
```

Comment:
Anonymizer script only supports a single keyspace when generating Gemini output.  This val is used to constrain that operation.